### ISSUE TITLE
is that possible to share spark environment between different test suites/files in a project

### ISSUE THREAD

**Issue by hangelwen** [holdenk/spark-testing-base#203]
With spark-testing-base, we can share spark env between tests in a test suite. That's great. But one could have many test classes that can be tested using the same local spark setup.  Preparing a spark env for each test class is still quite slow. The build process becomes quite slow for a large project. Is that possible to share a spark environment between different test classes?

**Comment by holdenk** [comment]
That support has been added experimentally in 0.7.3 (and fixed a bit in 0.7.4), but only for Spark 2.0 and higher.

**Comment by holdenk** [comment]
To enable re-using SparkContext/SparkSessions between tests add:
```
  // re-use the spark context
  override implicit def reuseContextIfPossible: Boolean = false
```
to your test suite.

Note: This is not recommend for Spark Version less than 2.2 and you should use the latest version of spark-testing-base for this.
